File: piston_test.php
<?php

$url = "https://emkc.org/api/v1/piston/execute";

$curl = curl_init();

$src = "print('hello world!')";

$data = [
    "language : python3",
    "source :" .$src,
    // "stdin :".$stdin
];

// $data->language = "python3";
// $data->source = $src;

$dataInJSON = json_encode($data);

// curl_setopt($curl, CURLOPT_POST, $dataInJSON);

curl_setopt_array($curl, 
[
	CURLOPT_URL => $url,
	CURLOPT_RETURNTRANSFER => true,
	CURLOPT_FOLLOWLOCATION => true,
	CURLOPT_ENCODING => "",
	CURLOPT_MAXREDIRS => 10,
	CURLOPT_TIMEOUT => 30,
	CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
	CURLOPT_CUSTOMREQUEST => "POST",
	CURLOPT_POSTFIELDS => $dataInJSON
]);

$response = curl_exec($curl);
$err = curl_error($curl);

curl_close($curl);

if ($err)
{
	echo "cURL Error #:" . $err;
}
else 
{
	echo $response;
}

?>

File: piston_test.py
import requests
import json

url = 'https://emkc.org/api/v1/piston/execute'

src = """
# Find square root of real or complex numbers
# Importing the complex math module
import cmath

num = 1+2j

# To take input from the user
#num = eval(input('Enter a number: '))

num_sqrt = cmath.sqrt(num)
print('The square root of {0} is {1:0.3f}+{2:0.3f}j'.format(num ,num_sqrt.real,num_sqrt.imag))
"""
stdin = """

"""
data = {
    "language": "python3",
    "source": src,
    "stdin": stdin,
    "args": []
}

response = requests.request('POST', url, data=data)

print(response.text)
input("Press any key...")